moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -263,7 +263,24 @@ bool planning_scene_monitor::CurrentStateMonitor::haveCompleteState(const ros::D
   return result;
 }
 
-bool planning_scene_monitor::CurrentStateMonitor::waitForCurrentState(double wait_time) const
+bool planning_scene_monitor::CurrentStateMonitor::waitForCurrentState(const ros::Time t, double wait_time) const
+{
+  ros::WallTime start = ros::WallTime::now();
+  ros::WallDuration elapsed;
```

Comment:
explicitly initializing this to ``(0)`` rather than relying on the default constructor might make the code slightly easier to read